desk/app/groups.hoon

```diff
@@ -387,6 +387,8 @@
     ::  force leave
     (emit [%pass wire %agent dock %leave ~])
   ::
+  =.  cor
+    (emit [%pass /load %arvo %b %wait now.bowl])
```

Comment:
Consider a slightly more specific wire, like `/load/active-channels`. We may need to defer from `+on-load` for other things in the future, we'll want to disambiguate them.

And, shouldn't this be conditional on the 4-to-5 migration? ie `=?  cor  ?=(%4 -.old)` and putting this in between the relevant state transforms above.

Reply:
Done.

Reply:
With your changes, it still sends the `%wait` unconditionally on every `+on-load` call, instead of only when we migrate from 4 to 5.